server/pkg/server/watchstate/watchstate.go

```diff
@@ -109,6 +110,7 @@ func (w *WatchState) send(item *localnetv1.OpItem) {
 	if w.Err != nil {
 		return
 	}
+	metrics.Kpng_node_local_events.Inc()
```

Comment:
This should increment the counter per label, i.e. per node name. But I don't think we have access to the node name here, maybe we need to move this higher up?

Reply:
okay that works for now ig :)